CMakeLists.txt

```diff
@@ -16,3 +16,9 @@ add_subdirectory(src)
 add_subdirectory(icons)
 add_subdirectory(man)
 add_subdirectory(rc)
+
+find_package(LATEX)
```

Comment:
If you have no latex installation during compilation, you don't get these files. I.e. installing latex later would require a pdfpc re-compilation.

Reply:
> If you have no latex installation during compilation, you don't get these files. I.e. installing latex later would require a pdfpc re-compilation.

Right, but so is with e.g., video support if gstreamer is not present.